vendor/arbiter/arbiter.cpp

```diff
@@ -2655,6 +2665,17 @@ AZ::Config::Config(const std::string s, const std::string profile)
     , m_endpoint(extractEndpoint(s, profile))
     , m_baseUrl(extractBaseUrl(s, m_service, m_endpoint, m_storageAccount))
 {
+    const std::string sasString = extractSasToken(s);
+    if (!sasString.empty())
+    {
+        const auto params = split(sasString, '&');
+        for (const auto& param : params)
+        {
+            const auto kv = split(param, '=');
+            m_sasToken[kv.at(0)] = kv.at(1);
```

Comment:
This needs some hardening. I'd rather see the result of split() checked for size and then use `[]`, but with `at`  you have to catch an exception.